integration-tests/.dockerignore

```diff
@@ -1,3 +1,9 @@
 Dockerfile
+.dockerignore
```

Comment:
That was for build speed up when the integrationtest was copied inside the dockerimage. With the current approach to runtime mount the directory with the test. This file is obsolte I wll delete it.